Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -646,6 +647,36 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.15,
 )
 
+class UpgradeWorkflow_JMEAK8PFNano(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Nano' in step:
+            stepDict[stepName][k] = merge([{'--customise': 'PhysicsTools/NanoAOD/custom_jme_cff.PrepJMECustomAK8PFNanoAOD_MC'}, stepDict[step][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        return fragment=="TTbar_13" and ('2017' in key or '2018' in key)
```

Comment:
This class is asking for 2017 and 2018 but why you need Run2? Shouldn't we target Run3?